modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/ScalaAkkaClientCodegen.java

```diff
@@ -166,7 +166,6 @@ public void processOpts() {
         additionalProperties.put(CodegenConstants.INVOKER_PACKAGE, invokerPackage);
 
         supportingFiles.add(new SupportingFile("README.mustache", "", "README.md"));
-        supportingFiles.add(new SupportingFile("pom.mustache", "", "pom.xml"));
```

Comment:
@ex0ns should this change be reverted?

Reply:
I'm not sure, the goal is to compile for both 2.13 and 2.12 which is not possible/easy with maven.
I migrated this codegen to sbt.

Reply:
the next version is a patch release so removing this file seems to be a breaking change.

can we simply keep it and update the dependency versions?

Reply:
I've added the pom.xml back (as well as the pom.mustache) :ok_hand: